src/clic.adoc

```diff
@@ -1415,6 +1353,75 @@ The {ret} instruction does not modify the {cause}.{pil} field in {cause}.
 == ssclic S-mode CLIC extension
 The ssclic extension depends on the smclic extension.
 
+=== S-mode CLIC Register Access via Indirect CSR Access
```

Comment:
Can you rename this to "Indirect Access S-mode CSRs"?

Reply:
```suggestion
=== Indirect Access S-mode CSRs
```